wfe/web-front-end.go

```diff
@@ -201,20 +215,21 @@ func (wfe *WebFrontEndImpl) NewRegistration(response http.ResponseWriter, reques
 		return
 	}
 
-	body, key, err := verifyPOST(request)
+	body, key, err := wfe.verifyPOST(request)
 	if err != nil {
 		wfe.sendError(response, "Unable to read/verify body", http.StatusBadRequest)
 		return
 	}
 
-	var init core.Registration
+	var init, unmarshalled core.Registration
 	err = json.Unmarshal(body, &init)
 	if err != nil {
 		wfe.sendError(response, "Error unmarshaling JSON", http.StatusBadRequest)
 		return
 	}
+	init.MergeUpdate(unmarshalled)
```

Comment:
This seems wrong; you're copying data from `unmarshalled` which is still all defaults.